tools/BaseImageUpdate/baseImage.ps1

```diff
@@ -46,7 +46,7 @@ function Update-ARM-BaseImages
     Setup-BaseImage-Script
 
     # Get all the Dockerfile file location
-    $fileLocale = $(Get-ChildItem -Recurse -Filter "Dockerfile" )
+    $fileLocale = $(Get-ChildItem -Recurse -Filter "Dockerfile" | Where {$_.FullName -notlike "*generic-mqtt-tester*"})
```

Comment:
If you want to future proof it further you can introduce a collection of excluded names. I am fine with this though.